omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -43,19 +43,13 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
         }
 
     self.validateCondition = self.validateCondition || function () {
-        var concept ;
-        if($scope.concept.concept.uuid) {
-            concept = $scope.concept.concept;
-        } else {
-            concept = $scope.condition.concept;
+        if ($scope.condition.concept===null||$scope.concept.concept==null){
```

Comment:
Do we need these two comparisons here ? @jnsereko   have you tested these changes with a running instance of openmrs to see that all performs as required!

Reply:
think the above if statement checks if its a concept(by having a uuid) and assigns a **Concept** to the `concept `variable or **Condition** to the `concept `variable.  

I have tested this using our demo by editing the **addcondition.controller.js** file using chrome tools

Reply:
results 

https://user-images.githubusercontent.com/58003327/115373830-ea3df600-a1d4-11eb-85cd-1b30f4c0c786.mp4